specification/components/schemas/FetchServicesList.yaml

```diff
@@ -0,0 +1,193 @@
+title: Service search response
+type: object
+required:
+  - meta
+  - resourceType
+  - status
+  - mode
+  - entry
+properties:
+  meta:
+    type: object
+    required: 
+      - profile
+    properties:
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-FetchServices-List-1'
+          example: 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-FetchServices-List-1'
+  resourceType:
+    type: string
+    enum:
+      - List
+    example: 'List'
+  status:
+    type: string
+    enum:
+      - current
+    example: 'current'
+  mode:
+    type: string
+    enum:
+      - snapshot
+    example: 'snapshot'
+  emptyReason:
+    type: object
+    required:
+      - coding
+    properties:
+      coding:
+        type: array
+        minItems: 1
+        maxItems: 1
+        items:
+          type: object
+          required:
+            - system
+            - code
+            - display
+          properties:
+            system:
+              type: string
+              enum:
+                - http://hl7.org/fhir/ValueSet/list-empty-reason
+              example: 'http://hl7.org/fhir/ValueSet/list-empty-reason'
+            code:
+              type: string
+              enum:
+                - nilknown
+                - notasked
+                - withheld
+                - unavailable
+                - notstarted
+                - closed
+              example: 'nilknown'
+            display:
+              type: string
+              example: 'no results'
+  entry:
+    type: array
+    items:
+      title: Singular entry with service details
+      type: object
+      required:
+       - item
+       - extension
+      properties:
+        extension:
+          type: array
+          items:
+            type: object
+            required:
+              - extension
+              - url
+            properties:
+              extension:
+                type: array
+                items:
+                  anyOf:
+                    - title: Extension to supply display prominently indicator
```

Comment:
I think this extension contains more details other than the display prominently indicator, perhaps a title of "Extension to supply search specific details for a service" would make more sense? It might also be worth including a link for the FHIR profile here (https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ServiceSearch-ListItem-1)

Reply:
Ah sorry I'm looking at the wrong part section of the OAS file here, these title makes sense here but maybe it makes sense for a general description of the extension as a whole as mentioned in my above comment?

Reply:
Yep I'll add that title